metricbeat/module/elasticsearch/node_stats/_meta/fields.yml

```
@@ -312,4 +312,21 @@
               type: long
             - name: rejected.count
               type: long
-
+    - name: indexing_pressure
```

Maybe something like this?
```suggestion
    - name: indexing_pressure
       type: group
       fields:
           - name: memory
              type: group
              fields:
                  - name: current.combined_coordinating_and_primary.bytes
                    type: long
                    format: bytes
                    description: xxx
                 - name: total.combined_coordinating_and_primary.bytes
                    type: long
                    format: bytes
                    description: yyy
```

---

@kaiyan-sheng thank you! this was very helpful